daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/TestCLIParsing.scala

```diff
@@ -969,4 +969,15 @@ class TestCLIParsing {
       e.getMessage.contains("""Input Buffer: <?xml version="1.0" encoding="UTF-8"?>""")
     )
   }
+
+  @Test def test_includeDoesNotExist_Error(): Unit = {
+    val schema = path(
+      "daffodil-cli/src/test/resources/org/apache/daffodil/cli/cli_schema_invalid_include.dfdl.xsd"
+    )
+    runCLI(args"parse -s $schema") { cli =>
+      cli.sendLine("123", inputDone = true)
+      cli.expectErr("[error]")
+      cli.expectErr(s"Schema context: Location in ${schema.normalize()}")
+    }(ExitCode.UnableToCreateProcessor)
```

Comment:
I imagine we already have a test that tests for failing to resolve schema locations. Maybe we can just update the expected diagnostics to be a bit more specific so we can avoid adding another test?